### PR TITLE
Added 1 FKW talk as practice

### PR DESCRIPTION
**Pull request by osg-cat**

### CODE REVIEW

File: _data/people/fkw888.yml [hunk 1]
```diff
@@ -6,6 +6,13 @@ institution: University of California, San Diego
 website: https://www-physics.ucsd.edu/fac_staff/fac_profile/faculty_description.php?person_id=494
 photo: /assets/images/team/Frank-Wuerthwein.jpg
 presentations:
+  - title: "State of OSG"
+    date: 19 Mar 2019
+    url: https://indico.cern.ch/event/759388/contributions/3353296/attachments/1814351/2964744/osgahm19-20190319-am11-fkw-a.pdf
+    meeting: "HOW2019 (Joint HSF/OSG/WLCG Workshop)"
+    meetingurl: https://indico.cern.ch/event/759388/
+    location: Jefferson Lab
+    focusarea: osglhc
```
Comment (gordonwatts) [inline _data/people/fkw888.yml:15]: The proper tag is "osg" for the IRIS-HEP website. Since we have only one bit of the OSG in here...

**Comment by gordonwatts** [comment]
Fix that one bit and we can merge this!

**Review by gordonwatts — CHANGES_REQUESTED** [review]
See comments for requested changes (sorry, not used to the review interface).

File: _data/people/fkw888.yml (hunk 1 above)
Comment (osg-cat) [reply _data/people/fkw888.yml:15]: OK, I will change it. But note that I was following the instructions here: <http://iris-hep.org/docs/newteammember>. It lists `osglhc` as the only OSG option for `focusarea`.

Comment (gordonwatts) [reply _data/people/fkw888.yml:15]: Crap! You are right. I was basing it off some internal notes. I'd not realized that had been updated. Ok - we have to keep it osglhc then. Sorry!

Comment (osg-cat) [reply _data/people/fkw888.yml:15]: OK, reverted.

Comment (gordonwatts) [reply _data/people/fkw888.yml:15]: Great!

**Pull request merged**